packages/api/src/logger/index.ts

```diff
@@ -350,12 +350,13 @@ interface PrismaLoggingConfig {
  */
 export const handlePrismaLogging = (config: PrismaLoggingConfig): void => {
   const logger = config.logger.child({
-    prisma: { clientVersion: config.db['_clientVersion'] },
+    prisma: { clientVersion: Prisma.prismaVersion.client },
   })
 
-  config.logLevels?.forEach((level) => {
+  config.logLevels?.forEach((level: any) => {
```

Comment:
This use of "any" here is the workaround such that level can be some value other than "beforeExit"

Reply:
Think we can take this out now, less to change once the prisma fix is in.

Reply:
Done.